### ISSUE TITLE
BTstack support needed for iOS build (analogs don't work with CFA)

### ISSUE THREAD

**Issue by bronnel** [libretro/RetroArch#2338]
The latest iOS build from cydia repo is working perfectly fine on my iPad mini retina iOS 8.4 jailbroken with Taij, however input have some problems, the analogs with select/start buttons are not working , I'm using controllers for all 1.7.3 and btstack tweaks from cydia, previously Retroarch wasn't depending on Controllers for all , it was depending purely on btstack, so I guess if this implemented on the latest build those problems would be fixed, you can also connect multiple controllers. Twinaphex please make this dream come true for us.

**Comment by twinaphex** [comment]
Yeah the BTStack support is in need of some bug fixes, it got broken somewhere in-between the transition from iOS 6 to iOS 7.

I am aware that the original MFi spec is a bit limited and that it doesn't allow you to map all the buttons, unfortunately this is Apple's fault and we cannot really do much about it.

Anyway I will try to look into supporting native BTStack better without the need for MFi or Controllers 4 All altogether. Can't give you any ETA on that though.

**Comment by bronnel** [comment]
I appreciate that, that is very good news , now at least we know it will fixed in the future

**Comment by bronnel** [comment]
Is it possible for a person like me with zero dev knowledge to help in that matter, I would love to get involved,  I maybe able to learn so basic stuff

**Comment by kiwanowski** [comment]
any news regarding the fix? C4A doesnt support multiple controllers :/

**Comment by mk8itra1n** [comment]
Any news about the reicast core for iOS version of reicast? The yabause core for iOS allows for somewhat playable gameplay but it's very slow and has choppy audio. In some games the music doesn't work but the core is pretty amazing progress. Just wondering about any updates on these 2 cores if there are any. Thanks in advance
